src/Sylius/Bundle/CurrencyBundle/spec/Form/Type/CurrencyTypeSpec.php

```diff
@@ -46,6 +46,16 @@ function it_should_build_form_with_proper_fields(FormBuilder $builder)
         ;
 
         $builder
+          ->add('enabled', 'checkbox', Argument::any())
+          ->willReturn($builder)
```

Comment:
Should I do this on everything in this method? Seems like the right thing to do then.